fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/properties/NbtIdentifier.java

```diff
@@ -0,0 +1,38 @@
+package net.fabricmc.fabric.api.fluids.properties;
+
+public final class NbtIdentifier {
```

Comment:
Not sure what this is doing for a fluid api?

Reply:
used as an Id for fluid properties